packages/server/src/router/auth.router.ts

```diff
@@ -20,3 +24,13 @@ export const AuthRouter: RouterDefinition<typeof AuthApi> = {
     res.sendStatus(200)
   }
 }
+
+async function checkCredentials(email: string, password: string) {
+  const result = await db
+    .userCollection()
+    .findOne({ email: { $regex: email, $options: 'i' } })
```

Comment:
This is NOT SECURE! We'd have to escape everything that's a special character in RegEx and use start/end markers.
See https://stackoverflow.com/a/50633536/5519485

We could use full text search like this: https://stackoverflow.com/a/39184944/5519485

Another option is to include a serparate field with a lower/upper case version of the email address (https://stackoverflow.com/a/4441412/5519485). 

Since we should use an index anyways, full text search is probably the simplest solution.